### PR TITLE
[12.0][MIG][WIP] stock_picking_report_valued

### PR DESCRIPTION
**Pull request by antoniorubiopesol**

### CODE REVIEW

**Review by angelmoya — APPROVED** [review]
Tested on Runbot and code review

LGTM :+1:

File: stock_picking_report_valued/models/stock_move_line.py
Comment (rousseldenis) [inline stock_picking_report_valued/models/stock_move_line.py]: related_sudo is True by default. Please remove all entries

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Minor comment

File: stock_picking_report_valued/models/stock_move_line.py
Comment (angelmoya) [reply stock_picking_report_valued/models/stock_move_line.py]: @rousseldenis I did it

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review.

Don't hesitate to complete the migration using https://github.com/OCA/maintainer-tools#readme-generator.

You have to change manifest 'website' key at least. https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py

File: stock_picking_report_valued/readme/BUGTRACKER.rst
Comment (Change2improve) [inline stock_picking_report_valued/readme/BUGTRACKER.rst:4]: I think this file is not necessary, see https://github.com/OCA/maintainer-tools/tree/master/template/module

File: stock_picking_report_valued/readme/DESCRIPTION.rst
Comment (Change2improve) [inline stock_picking_report_valued/readme/DESCRIPTION.rst:8]: I think you should remove the first lines and just let the description. See https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/DESCRIPTION.rst

File: stock_picking_report_valued/readme/MAINTAINER.rst
Comment (Change2improve) [inline stock_picking_report_valued/readme/MAINTAINER.rst:12]: I think this file is not necessary, see https://github.com/OCA/maintainer-tools/tree/master/template/module

File: stock_picking_report_valued/readme/USAGE.rst
Comment (Change2improve) [inline stock_picking_report_valued/readme/USAGE.rst:8]: I think you should remove the last lines and just let the usage description. See https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/USAGE.rst

**Review by Change2improve — CHANGES_REQUESTED** [review]
Minor changes in /readme

**Comment by novawish** [comment]
Just asking: Can we add the computations of total amount on `stock.move` as well? (using the same method as `stock.picking`)

PS: Also, can we make all the fields stored?

**Comment by rousseldenis** [comment]
@antoniorubiopesol Do you plan to attend comments ?

**Comment by emagdalenaC2i** [comment]
@antoniorubiopesol You should take a look at #60 and #59 

And if you can't move forward with this migration, please tell it so someone else can finish it.

**Comment by pedrobaeza** [comment]
Superseeded by #63

**Pull request closed without merging**
